src/app/nft-drop-mgr/nft-drop-mgr.component.ts

```diff
@@ -47,10 +53,52 @@ export class NftDropMgrComponent implements OnInit {
 
   datasource: IDatasource<IAdapter<any>> = this.infiniteScroller.getDatasource();
 
-  constructor(private _globalVars: GlobalVarsService, private backendApi: BackendApiService) {
+  feedTabs = [NftDropMgrComponent.SHOWCASE_MANAGEMENT, NftDropMgrComponent.SHOWCASE_PREVIEW_TAB];
+  switchingTabs = false;
+
+  constructor(
+    private _globalVars: GlobalVarsService,
+    private backendApi: BackendApiService,
+    private router: Router,
+    private route: ActivatedRoute
+  ) {
     this.globalVars = _globalVars;
   }
 
+  showShowcaseManagementTab() {
+    return (
+      this.activeTab === NftDropMgrComponent.SHOWCASE_MANAGEMENT &&
+      this.posts.length > 0 &&
+      (!this.loading || this.loadingNewDrop)
+    );
+  }
+
+  showShowcasePreviewTab() {
+    return (
+      this.activeTab === NftDropMgrComponent.SHOWCASE_PREVIEW_TAB &&
+      this.posts.length > 0 &&
+      (!this.loading || this.loadingNewDrop)
+    );
+  }
+
+  _handleTabClick(tab: string) {
+    this.activeTab = tab;
+    this.router.navigate([], {
+      relativeTo: this.route,
+      queryParams: { feedTab: this.activeTab },
```

Comment:
let's not call the query param feedTab. I think we call it feedTab in the feed component and this could cause some issues when merging queryParams.